src/pages/EntityDetails/Model/ActionLogs/ActionLogs.tsx

```diff
@@ -102,6 +124,39 @@ export default function ActionLogs() {
 
   const applicationList = () => Object.keys(modelStatusData.applications);
 
+  const OutputSelector = ({
```

Comment:
This is defined inside the `ActionLogs` component which means it'll get reconstructed every time `ActionLogs` is rerendered. It'd probably be best in its own file (though I know at the moment this project sometimes defines more than one component per file).

Though you could probably just use `ContextualMenu` inside the row object as this component is not too large and doesn't have any state or logic to test in isolation.

Reply:
Makes sense thanks for the explanation, this is very helpful!